meta/applications/yc_s15.md

```diff
@@ -0,0 +1,402 @@
+# YC Application
+
+hackEDU's nonprofit YC application for S15.
+
+### Application
+
+**Organization name**
+
+hackEDU
+
+**Organization url, if any**
+
+https://hackedu.us
+
+**If you have an online demo, what's the url?**
+
+N/A
+
+**What will your organization do?**
+
+hackEDU is an international community of programming clubs. We think it's silly
+that students who love music can join band, students who love sports can join
+an athletic team, but students who want to learn to code have to go home and
+code alone. Our vision is that every high school in the country will offer
+programming clubs where students can build things (and learn to build things)
+with other like-minded students.
+
+As an organization, we do two things: (1) create and provide a club "playbook",
+a guide on building awesome programming clubs, and (2) grow, connect and manage
+our broader community.
+
+**Where do you live now, and where would the organization be based after YC?**
+
+SF. SF.
+
+**Email address we should contact you at**
+
+zach@hackedu.us
+
+**Phone number(s)**
+
+[redacted]
+
+**Please provide an email address for each founder in the organization,
+including yourself. Founders must have at least 10% equity in the company. We
+will send an email to each founder to fill out additional information about
+themselves.**
+
+* zach@hackedu.us
+* jonathan@hackedu.us
+
+**Please enter the url of a 1 minute unlisted (not private) YouTube video
+introducing the founders.**
+
+**Please tell us about an interesting project, preferably outside of class or
+work, that two or more of you created together. Include urls if possible.**
+
+**How long have the founders known one another and how did you meet? Have any
+of the founders not met in person?**
+
+We've known each other since September 2014 (6 months) and met after a mutual
+friend introduced Zach, who had just won PennApps, the University of
+Pennsylvania's hackathon, to Jonathan, who was a former lead organizer of
+PennApps.
+
+**How far along are you?**
+
+We ran a pilot last semester with 25 schools and reached 1000 students, 35% of
+whom are female (2x national average) and 18% of whom are minorities in tech
+(3x national average). At MHacks V and PennApps X, the two largest collegiate
+hackathons with >1000 attendees from schools including Stanford, MIT, and
+Harvard, half of the winning teams were led by high school students, all of
+whom are part of their local hackEDU chapter.
```

Comment:
* Wording is really bad.
* Perhaps break up into bullet points.
* How much of this is impact that we created?